awscli/compat.py

```diff
@@ -257,3 +273,28 @@ def _windows_shell_quote(s):
         # quoted.
         return '"%s"' % new_s
     return new_s
+
+
+def is_windows():
```

Comment:
I was mainly copying what was done in the `help` logic. I'm fine switching over to `os.name`.

Reply:
No particular reason. I originally had it as `platform.system` because it was more heavily used in the codebase. I can switch though to `sys.platform`